Components/Avatar/avatar.html

```diff
@@ -93,11 +93,50 @@
                 <p class="p-content">Profile Avatars can be created using the class 'avatar', inside the img tag.</p>
 
                 <!-- Profile Avatar -->
-                <div class="avatar">
+                <div class="avatar-container">
+                    <img class="avatar" src="https://bit.ly/3J94zSO">
+                    <img class="avatar" src="https://bit.ly/3ou569I">
+                    <img class="avatar" src="https://bit.ly/34xfbf4">
+
                 </div>
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/OO9PRpBci23GOiIms42o"
+                    style="width: 563px; height: 242px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+                <hr class="margin-bottom-1">
+
+                <!-- Size Avatar --><span class="sub-title">Size Avatar</span>
+                <p class="p-content">Size of the avatars can be changed by adding sm | md | lg class to the existing
+                    avatar class.</p>
+
+                <div class="avatar-container">
+                    <img class="avatar sm" src="https://bit.ly/3J94zSO">
+                    <img class="avatar md" src="https://bit.ly/3ou569I">
+                    <img class="avatar lg" src="https://bit.ly/34xfbf4">
 
+                </div>
+                <iframe class="iframeHide" src="https://carbon.now.sh/embed/Jqbhd1Saz6e0MEtgvZds"
+                    style="width: 588px; height: 242px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
                 <hr class="margin-bottom-1">
 
+                <!-- Square Avatar --><span class="sub-title">Square Avatar</span>
+                <p class="p-content">Adding a class 'square' to existing class, will generate a square shaped avatar.
+                </p>
+
+                <div class="avatar-container">
+                    <img class="avatar sm square" src="https://bit.ly/3J94zSO">
+                    <img class="avatar md square" src="https://bit.ly/3ou569I">
+                    <img class="avatar lg square" src="https://bit.ly/34xfbf4">
```

Comment:
add size classes to the end.

Reply:
I have added the sizes to the end . Thanks for reviewing